### DISCUSSION TITLE
Network Map to Highlight Online Devices in Green

### DISCUSSION THREAD

**Discussion by anjota80** [netalertx/NetAlertX#616] [Ideas]
The network map is a great feature within Pi.Alert. It would be nice to see the online devices highlighted in green including the un-discoverable devices. Currently, the network map only highlights devices offline, in red.

![2024-04-02_22-48-48](https://github.com/jokob-sk/Pi.Alert/assets/53611664/5ec43d2b-ad70-44a1-bd93-5d27af94ee37)

**Comment by jokob-sk** [comment]
Hi there,

The grey color indicates online devices, so everything that is not red is online. 

I tried using the green color but it introduces too much cognitive load into the design. Because of this, I'm leaning towards keeping the design as it is. 

Hope that makes sense,
j

**Reply by anjota80** [reply]
Yes, I understand. Thanks for looking.
